tools/cli/pkg/ers/client/http.go

```diff
@@ -0,0 +1,86 @@
+package client
+
+import (
+	"context"
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	"github.com/kyma-project/control-plane/tools/cli/pkg/ers"
+	"github.com/kyma-project/control-plane/tools/cli/pkg/logger"
+	"github.com/pkg/errors"
+)
+
+const timeoutInMilli = 400
```

Comment:
400 Millisecond might be a bit optimistic as larger data might need longer to transmit

Reply:
I would say even more than 3 seconds